### PR TITLE
Leader board practice project with webpack and git flow 

### PR DESCRIPTION
**Pull request by juaryimami**
### In this project:-
---
-  I set up a JavaScript project for the Leaderboard list app, using webpack and ES6 features, notably modules. 
- I have developed a first working version of the app following a wire frame.
- Thank your for reviewing my project!!!.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,84 @@
-# leaderboard-project
\ No newline at end of file
+# leaderboard-project
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+In this project, I set up my To-Do List web page. My goal was add the functionality to add and display each todo task inside my task list.
```
Comment (KDlamini) [inline README.md]: Kindly update your description and make it relevant to this project.

File: README.md [hunk 2]
```diff
@@ -1 +1,84 @@
-# leaderboard-project
\ No newline at end of file
+# leaderboard-project
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+In this project, I set up my To-Do List web page. My goal was add the functionality to add and display each todo task inside my task list. 
+
+![screenshot](/public/Screenshot%20from%202022-10-06%2009-06-02.png)
```
Comment (KDlamini) [inline README.md]: Kindly update your screenshot or otherwise remove it

File: README.md [hunk 3]
```diff
@@ -1 +1,84 @@
...
+You can view my hosted website in the link shown below.
+GitHub: [@JUPAC](https://juaryimami.github.io/todo-list-app/dist/)
+
+## :hammer: Built With
+
+- HTML, CSS, JS and Webpack.
+
+To get a local copy up and running follow these simple steps:
+
+1. Go to the [repository page]( https://juaryimami.github.io/todo-list-app/).
```
Comment (KDlamini) [inline README.md]: Kindly update your Github repository links...

File: README.md [hunk 4]
```diff
@@ -1 +1,84 @@
...
+   - To get a local copy up and running follow these simple example steps.
+
+## Prerequisites
+   - please install git and node.js in your computer.
+   - Also install a code editor.
+   - A web browser to view the output.
+
+## Setup
+   - To clone my repository run this command `https://github.com/juaryimami/todo-list-app.git`  
+
```
Comment (KDlamini) [inline README.md]: Same here...

File: README.md [hunk 5]
```diff
@@ -1 +1,84 @@
...
+👤 **JuharSeid**
+
+- GitHub: [@JuharYimer](https://github.com/juaryimami)
+
+## 🤝 :raised_hand: :raised_hand: Contributions
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/juaryimami/todo-list-app/issues).
+
```
Comment (KDlamini) [inline README.md:73]: Kindly update your issues link and make it relevant to this project.

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @juaryimami 🙋‍♀️,  

Great work so far! 💯 
There are some issues that you still need to work on to improve your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 

✔️  Webpack configured correctly
✔️  Html follows the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png) 
✔️  Application runs as expected
✔️  No linter errors

## Required Changes ♻️
 
 - [x] Please update your README to make it relevant to this project. Your description, screenshot, links, and references are all for the Todo list project.
 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

- [ ] The proper way to follow GitFlow is to make the `develop` branch the default branch. Then introduce every new feature branch to the development branch. The new pull request should be merging the new branch to the `develop` branch as explained in the [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md) documentation.

image

- This means that you do not merge the `develop` branch to the `main` branch until the entire project is complete. In the future, kindly consider using different branches for every new task, and they should all be merged into the `develop` branch until the final stage of development.


Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (juaryimami) [reply README.md]: ok ok thank your so much I forgot to update.

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @juaryimami   , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct GitHub Flow.
✔️ Professional   README
✔️ Good commit messages

almost

## Required Changes ♻️

- [x] As stated in the project requirement and suggested by the previous reviewer you didn;t follow the correct [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).Please  make your development branch [the default branch](https://docs.github.com/en/github/administering-a-repository/managing-branches-in-your-repository/changing-the-default-branch#changing-the-default-branch) and create a new feature branch, After that, add changes in a feature branch(es). You will need to create a pull request to compare your feature branch with the development branch and submit the link to that pull request via your Dashboard. 
_Check the comments below the review._

## Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by juaryimami** [comment]
kindly sir your are blocking my approval. I used git flow release branch isn't it part of git flow. it is not fair. Making my development branch to default is optional. it is up to me.

**Comment by Abenezer-Tilahun** [comment]
Hello @juaryimami  for this project you have to follow the Git-flow, I am not blocking your approval brother .For future Lead board project activity you should create pull request from the feature branch to development branch(default branch) , After you finished Lead board project you will merged the result to main branch. I am  not here discourage you but it helps you for future project, Please update it .

**Comment by juaryimami** [comment]
`git flow release start '0.1.0'` isn't it a good git flow practice bro.

**Comment by juaryimami** [comment]
after making updates the I merge to my devleopment ranch by
`git flow release finish '0.1.0'` this is what i do.

**Comment by Abenezer-Tilahun** [comment]
Hello @juaryimami  I understand you brother and you are right. but it's stated in one of the project requirement that why i suggested you to make changes but at this moment request for reviewer and the reviewer will approve it ,still in the future i recommend you to follow the project requirement .

**Comment by juaryimami** [comment]
ok thanks bro

**Review by uwadonat — APPROVED** [review]
## Hi @juaryimami,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
### Highlights
- Good communication with reviewers ✔️
- No linter erros ✔
approval
️
# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**